specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2022-04-01-preview/apimanagement.json

```diff
@@ -238,6 +238,39 @@
       "pattern": "^[^*#&+:<>?]+$",
       "x-ms-parameter-location": "method"
     },
+    "AuthorizationProviderIdParameter": {
```

Comment:
>AuthorizationProviderIdParameter [](http://example.com/codeflow?start=5&length=32)

Is this API version in public preview or private preview?

Reply:
@rkmanda It's in public preview